build-system/amp.extern.js

```diff
@@ -30,6 +30,12 @@ window.context.experimentToggles;
 // Service Holder
 window.services;
 
+// Safeframe
```

Comment:
These externs don't apply to AMP in general, except for the integration.js and ampcontext-lib.js binaries. Could you add a note that they should be extracted to a separate externs file?